debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -866,6 +866,10 @@ protected String tableWhitelist() {
     protected String columnBlacklist() {
         return getConfig().getString(COLUMN_BLACKLIST);
     }
+    
+    protected String keyMapper() {
```

Comment:
Could you push this up to `RelationalDatabaseConnectorConfig`, passing the value via the constructor super call?

Reply:
Well I would prefer not to as it would break histo & all the other connectors. And why this would be useful ?

Reply:
We try to unify as much as possible across the different (relational) connectors. So whatever can be done once in `RelationalDatabaseConnectorConfig`, should be. I can take care of it too when merging it.

Reply:
I've pushed one more commit to this PR to address this.

Reply:
@gunnarmorling : thanks. Nice small improvement ;-)